### ISSUE TITLE
github-releases (ideally with apk)

### ISSUE THREAD

**Issue by DJCrashdummy** [PhilippC/kp2a_keyboardswap#2]
since kp2a is still not available via f-droid and non of your plug-ins, is it possible to also release this plug-in with apks on github?

thanks for considering publishing without google!

**Comment by DJCrashdummy** [comment]
@PhilippC is this possible? please! :pray:

**Comment by DJCrashdummy** [comment]
as this app got recently an update, i'm kindly and imploringly asking you @PhilippC again to please release an apk at github, that users without a google account can easily install it resp. even @izzysoft can pick it up automatically and provide it (and its updates) via [his fdroid-repo](https://gitlab.com/IzzyOnDroid/repo).

**Comment by IzzySoft** [comment]
Looks like your *nonet is looking for its sibling in my repo :wink:

**Comment by DJCrashdummy** [comment]
@IzzySoft i know the "normal" version exceeds your storage (and personally i prefer the *nonet version anyways :wink:). this repo/issue is about the little but pretty handy *KeyboardSwap Plugin*, which swaps your keyboards automatically.

**Comment by PhilippC** [comment]
here you go: https://github.com/PhilippC/kp2a_keyboardswap/releases/tag/1.0.5

**Comment by IzzySoft** [comment]
@DJCrashdummy I know. Read my previous comment again: I said that one wants its (little) sibling. Oh, I see where the misunderstanding was: I missed the term "little" so you thought I meant the "big" – but that would have been the wrong repo then :smile:

@PhilippC Thanks! Added, will show up tomorrow [here](https://apt.izzysoft.de/fdroid/index/apk/keepass2android.plugin.keyboardswap2). I've set the same LiberapayID as for *nonet, hope that's fine? And while on it, I've added your Github Sponsors page as donation link to both.

PS: Funnily, GMS is reported for the plugin as well. I've checked the Smali structs, and this time there's just a single obfuscated candidate: `./smali/crc64546702c72fe163a8` (again, no "clear-name mention" of any GMS path). Hope this helps to pin it down.
